netlink-request/README.md

```diff
@@ -0,0 +1,5 @@
+# `netlink-request`
+
+A helper to perform requests using the Linux netlink interface.
+
+Versioning is held in lockstep with innernet, although this may change in the future.
```

Comment:
IIUC, it would make sense to edit https://github.com/tonarino/innernet/blob/main/release.sh to handle `netlink-request/Cargo.toml` the same way as `wireguard-control/Cargo.toml` to ensure the version is locked in step.

Reply:
+1 to this being the easiest direction to take it.

As a future thought, these libraries are less married to the development of innernet directly, so there may be more cases in the future where you'd want to push a fix for `netlink-request` without needing to update the innernet semver at the same time.

Reply:
Good catch, added to `release.sh` in 3703001, I've manually checked that the `perl` command works.

On a slightly unrelated note, I think `cargo release` can now supply many duties we do manually in `release.sh` (replacing versions, committing, tagging). I see we have that disabled in `release.toml`. At some point we may revisit that.